lua/ulx/modules/sh/cfc_cheese.lua

```diff
@@ -0,0 +1,29 @@
+CFCUlxCommands.Cheese = CFCUlxCommands.Cheese or {}
+local cmd = CFCUlxCommands.Cheese
```

Comment:
All the other commands have the subtable in camel case, if we wanna take that, it should be it's own PR.
Keep it camel for now
```suggestion
CFCUlxCommands.cheese = CFCUlxCommands.cheese or {}
local cmd = CFCUlxCommands.cheese
```